synapse/lib/stormtypes.py

```diff
@@ -3400,6 +3418,9 @@ async def iter(self):
     def value(self):
         return self.info.pack()
 
+    def ismutable(self):
```

Comment:
This should simply be an inhertied implementation that returns defaults to returning a class local that is True by default.